docs/content/preview/manage/backup-restore/snapshot-ysql.md

```diff
@@ -31,79 +31,102 @@ showAsideToc: true
 
 </ul>
 
-YugabyteDB supports distributed backup and restore of YSQL databases. Backing up and restoring of individual tables within a database is not yet supported.
+The most efficient way to backup the data stored in YugabyteDB is to create a distributed snapshot. A snapshot is a consistent cut of a data taken across all the nodes in the cluster. For YSQL, snapshots are created on per-database level. Backing up individual tables is currently not supported.
 
-## Create a snapshot for a YSQL database
+When YugabyteDB creates a snapshot, it doesn't physically copy the data, but instead creates hard links to all the relavant files. These links reside on the same storage volumes where the data itself is stored, which makes both backup and restore operations nearly instantanious.
 
-1. Get the current YSQL schema catalog version by running the following [`yb-admin ysql_catalog_version`](../../../admin/yb-admin/#ysql-catalog-version) command.
+{{< note title="Note on space consumption" >}}
 
-    ```sh
-    yb-admin -master_addresses <ip1:7100,ip2:7100,ip3:7100> ysql_catalog_version
-    ```
+There are no technical limitations on how many snapshots you can create. However, it's important to keep in mind that by increasing the number of snapshots stored, you also increase the amount of space required for the database. The actual overhead depends on the workload, so we recommend to estimate it by running tests based on your applications.
```

Comment:
Any recommendations on what tests to run, and how, or is that going to be super-specific to each use case?
```suggestion
There are no technical limitations on how many snapshots you can create. However, increasing the number of snapshots stored also increases the amount of space required for the database. The actual overhead depends on the workload, but you can estimate it by running tests based on your applications.
```